src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationParameters.cs

```diff
@@ -147,17 +155,17 @@ private static async Task<AuthenticationParameters> CreateFromResourceUrlCommonA
 
             try
             {
-                IHttpClient request = new HttpClientWrapper(resourceUrl.AbsoluteUri, null);
-                await request.GetResponseAsync().ConfigureAwait(false);
+                OAuthClient client = new OAuthClient(HttpManager, resourceUrl.AbsoluteUri, null);
+                await client.ExecuteRequestAsync<IHttpWebResponse>().ConfigureAwait(false);
```

Comment:
Why do I need to pass a type as a generic param ? i expect that ExecuteRequestAsync to always return IHttpWebResponse.